addons/fortify/functions/fnc_deployObject.sqf

```diff
@@ -70,6 +70,7 @@ private _mouseClickID = [_player, "DefaultAction", {GVAR(isPlacing) == PLACE_WAI
             [_unit, _object] call FUNC(deployConfirm);
         } else {
             TRACE_1("deleting object",_object);
+            [QGVAR(onDeployStop), [_unit, _object, _cost]] call CBA_fnc_localEvent;
```

Comment:
Not sure whether `typeOf _object` is more appropriate, given that _object is deleted in the next line.
On the other hand, it would allow 3rd party hooks to run code on object before it's deleted.

Reply:
I feel like `typeOf _object` is more fitting because the object is being deleted on the next line anyway. It seems almost misleading to pass the object to an event handler, but delete it right after emitting the event.

Reply:
We can delay the deleting of the object by a frame if necessary, I don't think it would pose too much of an issue.

The reason why `_object` is more interesting than `typeOf _object` is because the former can be derived easily from the prior and you might want to get e.g. `vectorUp` or other measurements related to the physical object's placement in the world.

@LinkIsGrim I need input on this.

Reply:
I'm not a fan of delaying object deletion. All eventhandlers for `QGVAR(onDeployStop)` will execute before the object is deleted.

Reply:
So we can leave it as is?

Reply:
Yes